### DISCUSSION TITLE
Latest episodes for podcast in Android Auto

### DISCUSSION THREAD

**Discussion by osimon72** [advplyr/audiobookshelf#3759] [Ideas]
Hi, 
I very much like the server and the android app. Unfortunately I am missing the "latest episodes" within Android Auto. This is my standard view on the device, but not in Android Auto. Therefore I have to browse through the podcasts, whcih takes a lot of time.

Would be great to have that feature available.

**Comment by Vito0912** [comment]
Iirc this is already added in the PR https://github.com/advplyr/audiobookshelf-app/pull/1322
Idk the current status of it when/if it's getting merged, but it lists:

> Recent books, series, authors, podcasts and episodes shelves are listed under Recent tab.

**Reply by osimon72** [reply]
Sorry, dont get it. According your info, it should be implemented.
After updating the server and the android app, I still dont have anything else than the library in Android Auto.
Used version was 2.17.6

Thanks

**Reply by Vito0912** [reply]
Ah sorry. There was a misunderstanding.

This feature exists as a pull request, which means it needs to be merged (accepted) into the main app first.

Until the pull request is accepted, this feature will not be available in the app.

**Reply by BCollins521** [reply]
Which pull request is this tied to?

**Reply by Vito0912** [reply]
@BCollins521 I linked it in the threads first comment

**Reply by BCollins521** [reply]
Hmm maybe I missed it when I was looking cause most of that request mentions book related fixes/requests instead of podcasts
